test/tests.js

```diff
@@ -0,0 +1,15 @@
+import('./tests/p5.Master.js');
```

Comment:
it simple and clean , now we have to write tests in `tests` folder and then import it here!